distributionviewer/core/static/js/app/components/containers/chart-config-container.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import ChartConfig from '../views/chart-config';
+import * as metricApi from '../../api/metric-api';
+
+
+class ChartConfigContainer extends React.Component {
+  componentDidMount() {
+    metricApi.getMetrics();
```

Comment:
This isn't your fault, but the way I wrote `getMetrics`, it dispatches both the metadata (the output of `/metrics/`) and the data (the output of each `/metric/#/`). We only need the metadata here, so we could refactor the API to allow for that. I'll follow up with a PR after this lands.